doc/userguide/lua/libs/base64.rst

```diff
@@ -0,0 +1,45 @@
+Base64
+------
+
+Base64 functions are expose to Lua scripts with the
+``suricata.base64``. For example::
+
+  local base64 = require("suricata.base64")
+
+Functions
```

Comment:
does it make sense to expose the various modes of suri base64 decoding explicitly?

Reply:
We expose the pad and no pad options, and a padding optional decode. That leaves the 2 RFC modes that I honestly see as more part of mime.  But it is trivial to add:
- `decode_rfc2045`
- `encode_rfc2045`
- etc...

Reply:
Added.